pkg/cli/admin/release/info.go

```diff
@@ -775,6 +775,33 @@ func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*Relea
 				errs = append(errs, err)
 				return true, nil
 			}
+			// Try the registry/repo passed from the user first.  If image does not exist,
+			// proceed with the release info from the release image-references, from readReleaseImageReference above
+			for _, tag := range is.Spec.Tags {
+				// this is user-passed image name
+				userGivenImage, err := imagereference.Parse(image)
+				if err != nil {
+					errs = append(errs, err)
+					return true, nil
+				}
+				// Digest is the sha of each component in the release image-reference.
+				// If can't get digest, skip this tag, this happens when user has
+				// replaced a component in the release with a new image.
+				digest, err := imagereference.GetSpecDigest(tag.From.Name)
+				if err != nil {
+					continue
+				}
+				// try to verify user-passed image tag first, in case of mirrored images this will
+				// exist, and if so, use this instead of the release image-reference. If this doesn't
+				// exist, proceed with tag from release image-reference from the readReleaseImageReference above
+				givenTag := fmt.Sprintf("%s@%s", userGivenImage.AsRepository(), digest)
```

Comment:
tag is confusing, since this is more `imageID`, `imageDigest`, right?

Reply:
I'm a bit confused, you're creating a new reference from user-passed registry and release's image SHA, right? And then you parse it once again in `verifyImageExists`. If this works you replace the tag contents, correct?

Reply:
I still don't understand why you can't just use the `DockerImageReference` returned from `userGivenImage.AsRepository()` invocation and set digest there. Iow.
```go
newRef := userGivenImage.AsRepository() // this will clear both Tag and ID fields
newRef.ID = digest
if verifyImageExists(opts, newRef)...
```
This way `verifyImageExists` will work with already passed reference, so there's no need for double parsing.

Reply:
thanks! updated, now passing imagereference.DockerImageReference to verifyImageExists, and if found, `tag.From.Name = newRef.String()`